birdhouse/components/finch/service-config.json.template

```diff
@@ -20,6 +24,11 @@
       "rel": "service-desc",
       "type": "text/xml",
       "href": "https://${PAVICS_FQDN_PUBLIC}${TWITCHER_PROTECTED_PATH}/finch?service=WPS&request=GetCapabilities"
+    },
+    {
+      "rel": "service-meta",
+      "type": "application/vnd.oci.image.index.v1+json",
+      "href": "${FINCH_IMAGE_URI}"
```

Comment:
If a new var is used for template expansion it should be added to `VARS` or `OPTIONAL_VARS` in the `default.env` of the corresponding component.  Otherwise, template expansion will not work.

I guess we do not validate the value in "href", that's why we did not caught the wrong template expansion?

Reply:
Optional vars added. See other comments for the validation.